envs/example/ci-full/group_vars/all.yml

```diff
@@ -16,13 +29,46 @@ neutron:
   lbaas:
     enabled: True
 
+common:
```

Comment:
Trying to under why many of things not in envs before needs to be added to lvm enablement?